### PR TITLE
Issue #Mentor : fix : api issue fix and network added

### PR DESCRIPTION
**Pull request by pradeepkumarcm96**

### CODE REVIEW

File: src/app/core/services/http/http.service.ts [hunk 1]
```diff
@@ -15,13 +22,17 @@ export class HttpService {
   constructor(
     private http: HTTP,
     private userService:UserService,
-    private toastService:ToastService
+    private network:NetworkService,
+    private toastService:ToastService,
+    private loaderService: LoaderService,
+    private localStorage: LocalStorageService,
+    private injector: Injector
```
Comment (kiranharidas187) [inline src/app/core/services/http/http.service.ts:29]: rm unwanted imports

File: src/app/core/services/network.service.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  public netWorkCheck() {
+    this.getCurrentStatus();
+    this.disconnectSubscription = this.network.onDisconnect()
+      .subscribe(() => {
+        this.isNetworkAvailable = false;
+      });
+    this.connectSubscription = this.network.onConnect().subscribe(() => {
+      this.isNetworkAvailable = true;
+    });
+    console.log(this.isNetworkAvailable);
```
Comment (kiranharidas187) [inline src/app/core/services/network.service.ts:24]: rm

File: src/app/core/services/network.service.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  }
+
+
+  public getCurrentStatus() {
+    if (this.network.type == 'none') {
```
Comment (kiranharidas187) [inline src/app/core/services/network.service.ts:29]: use ternary operator

File: src/app/core/services/http/http.service.ts (hunk 1 above)
Comment (pradeepkumarcm96) [reply src/app/core/services/http/http.service.ts:29]: all imports are needed

**Pull request merged**